tests/unit/test_configuration.py

```diff
@@ -110,12 +108,11 @@ def test_print_users(self):
 
         f = io.StringIO()
 
-        try:
+        with pytest.raises(SystemExit):
             with contextlib.redirect_stdout(f):
                 conf.print_users()
```

Comment:
```suggestion
        with pytest.raises(SystemExit) as excinfo:
            with contextlib.redirect_stdout(f):
                conf.print_users()
        assert excinfo.value.code == 0
```

Can you try something like this to assert the error code of the `SystemExit` instance?

The pytest API can allow you to access the instance of the exception, you can see [details docs here](https://docs.pytest.org/en/latest/how-to/assert.html#assertions-about-expected-exceptions).

Reply:
Done. Don't know why I added error code assertion in other places but not this file lol..